test/testenv/verificationutils.go

```diff
@@ -504,3 +505,37 @@ func VerifySplunkSecretViaAPI(deployment *Deployment, testenvInstance *TestEnv,
 		}
 	}
 }
+
+// VerifyPVC verifies if PVC exists or not
+func VerifyPVC(deployment *Deployment, testenvInstance *TestEnv, ns string, pvcName string, expectedToExist bool, verificationTimeout time.Duration) {
+	gomega.Eventually(func() bool {
+		result := false
+		pvcExists := false
+
+		pvcsList := DumpGetPvcs(testenvInstance.GetName())
+		for i := 0; i < len(pvcsList); i++ {
+			if strings.EqualFold(pvcsList[i], pvcName) {
+				pvcExists = true
+				break
+			}
+		}
```

Comment:
This can be simplified by replacing line 522-529 by below : 
```
testenvInstance.Log.Info("PVC Status Verified",  "PVC", pvcName, "STATUS", pvcExists, "EXPECTED", expectedToExist)
return result
}, verificationTimeout, PollInterval).Should(gomega.Equal(expectedToExist))